sympy/concrete/summations.py

```diff
@@ -512,6 +512,21 @@ def is_convergent(self):
         except NotImplementedError:
             pass
 
+        ### ---------- Raabe's test -------------- ###
+        if lim_ratio == 1:  # ratio test inconclusive
```

Comment:
There was a discussion in the other pull request about whether or not this is needed.

Reply:
imo it is necessary...

Reply:
Why is it necessary?

Reply:
Are you talking about raabe's test or the condition??

Reply:
The condition. Your answer above says "imo it is necessary". I am sure that I agree but I would prefer it if you write an explicit reason (an opinion is not a reason).

Reply:
We implement Raabe's test only after ratio test fails, and the condition when ratio test fails is when it is 1, so definitely it is necessary, otherwise how will we come to know when to implement raabe's test.